app/controllers/Application.java

```diff
@@ -1797,44 +1802,43 @@ public static Result getTuningParameter(String jobId) {
           .order()
           .desc(TuningJobDefinition.TABLE.createdTs)
           .findUnique();
-      logger.info("JobSuggestedParamSet:: " + jobSuggestedParamSet.id);
+      logger.info("Respective JobSuggestedParamSet: " + jobSuggestedParamSet.id);
       if (userSuggestedParamSet == null) {
-        logger.info("No user suggested param set for jobDefinitionId:" + jobDefinitionId);
+        logger.info("No user suggested param set for jobDefinitionId: "  + jobDefinitionId);
       } else {
         logger.info("User suggested param set: " + userSuggestedParamSet.id);
       }
       logger.info("Tuning Job Definition Id: " + tuningJobDefinition.job.id);
       Boolean autoApply = tuningJobDefinition.autoApply;
       TuningAlgorithm tuningAlgorithm = jobSuggestedParamSet.tuningAlgorithm;
       List<TuningParameter> parametersList = getTuningParametersListForJob(tuningAlgorithm.id);
-      logger.info("Tuning Algorithm id: " + tuningAlgorithm.id);
```

Comment:
Why this line is removed ?

Reply:
This log was not that necessary as Tuning Algorithm id can be fetched from TuningJobDefinition at the time of debugging, so removed it considering as redundant.